docusaurus/docs/get-started/get-started.mdx

```diff
@@ -15,6 +15,28 @@ import ScaffoldNPM from '@snippets/createplugin-scaffold.npm.md';
 import ScaffoldPNPM from '@snippets/createplugin-scaffold.pnpm.md';
 import ScaffoldYarn from '@snippets/createplugin-scaffold.yarn.md';
 
+# Grafana plugin developer's guide
+
+Extend Grafana's built-in capabilities with plugins. Plugins enable Grafana 
+to accomplish specialized tasks, custom-tailored to your requirements. 
+By making a plugin for your organization, you can connect Grafana to other 
+data sources, ticketing tools, and CI/CD tooling.
+
+You can create plugins for private use or contribute them to the open 
+source community by publishing to the Grafana 
+[plugin catalog](https://grafana.com/plugins). 
+This catalog has hundreds of other community and commercial plugins.
+
+## Types of plugins
+
+You can create several types of plugins, including:
+
+* **Panel plugins** - Visualize data and navigate between dashboards.
+* **Data source plugins** - Link to new databases or other sources of data.
+* **App plugins** - Create rich applications for custom out-of-the-box experiences.
+
+## Plugin tools
```

Comment:
we could extend below the content here, before the quick start to elaborate. Something like this:

"This guide is designed to help you quickly get up and running with a working plugin. From here you can explore the rest of our documentation and [examples](https://github.com/grafana/grafana-plugin-examples/) to build your knowledge and add more advanced capabilities."

Reply:
In my revision I have removed the section on types of plugins. It's possible we may want to add a new topic "Types of plugins" to the Introduction section at some point, but for now I think the short descriptions we give lower on the page are fine.